finder-app/manual-linux.sh

```diff
@@ -1,17 +1,21 @@
 #!/bin/bash
 # Script outline to install and build kernel.
-# Author: Siddhant Jajoo.
+# Author: Siddhant Jajoo 
+# Modified by: Krish Shah
 
 set -e
 set -u
 
 OUTDIR=/tmp/aeld
 KERNEL_REPO=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git
-KERNEL_VERSION=v5.1.10
+KERNEL_VERSION=v5.15.163
 BUSYBOX_VERSION=1_33_1
 FINDER_APP_DIR=$(realpath $(dirname $0))
 ARCH=arm64
 CROSS_COMPILE=aarch64-none-linux-gnu-
+YYLLOC_FIX_COMMIT_HASH=e33a814e772cdc36436c8c188d8c42d019fda639
```

Comment:
This shouldn't be necessary anymore, the 5.15 kernel should have the fix for yyloc errors.  See the tags at https://github.com/gregkh/linux/commit/e33a814e772cdc36436c8c188d8c42d019fda639 which include v5.15.3